check_test.go

```diff
@@ -48,7 +48,7 @@ func TestEncodeDomainCheck(t *testing.T) {
 func TestEncodeDomainCheckLaunchPhase(t *testing.T) {
 	con := &Conn{}
 	con.Greeting.Extensions = []string{ExtLaunch}
-	err := con.encodeDomainCheck([]string{"hello.com", "foo.domains", "xn--ninja.net"}, "claims")
+	err := con.encodeDomainCheck([]string{"hello.com", "foo.domains", "xn--ninja.net"}, map[string]string{"launch:phase": "claims"})
```

Comment:
I tried removing `map[string]string`, but Go didn't like that. Is there an alternate shorthand for map literals?